tests/test_uncrustify/failure.py

```diff
@@ -56,3 +56,15 @@ def __init__(self, expected, actual):
     def __str__(self):
         return 'Output {!r} does not match expected output {!r}'.format(
             self.actual_path, self.expected_path)
+
+# =============================================================================
+class UnexpectedlyPassing(Failure):
```

Comment:
Pedantic: this is still a test "failure", "UnexpectedlyPassingFailure" might be better :slightly_smiling_face:, even if it *is* a mouthful.

Reply:
I had a second thought exactly about this. Sure, let's be pedantic.

Reply:
:smile: